hack/update-go-workspace.sh

```diff
@@ -37,6 +37,12 @@ rm -f go.work go.work.sum
 
 # Generate the workspace.
 go work init
+(
```

Comment:
`sed -i -e '1i\// This is a generated file. Do not edit directly.\n' go.work` may be simpler? but this is good as well!